packages/embeds/embed-core/src/embed.ts

```diff
@@ -116,6 +116,31 @@ function validate(data: Record<string, unknown>, schema: ValidationSchema) {
   }
 }
 
+function getColorScheme(el: HTMLElement) {
+  const pageColorScheme = getComputedStyle(el).colorScheme;
+  if (pageColorScheme === "dark" || pageColorScheme === "light") {
```

Comment:
Initially handle `colorScheme` of dark and light only. Theoretcially there can be more color-schemes but let that scenario come up in production and handle the right way then.